yew-macro/tests/macro/html-list-fail.stderr

```diff
@@ -15,10 +15,10 @@ error: this closing fragment has no corresponding opening fragment
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
 
 error: this opening fragment has no corresponding closing fragment
- --> $DIR/html-list-fail.rs:6:13
+ --> $DIR/html-list-fail.rs:6:15
   |
 6 |     html! { <><> };
-  |             ^^
+  |               ^^
```

Comment:
Looks like this is using the `close` span for some reason

Reply:
Well, it kind of makes sense that it would mark the second one, doesn't it? You've essentially opened two fragments, and it should really be looking for the closing fragment of the "inner" one first. Right?

Reply:
Ah right, yeah that does make more sense!